Vagrantfile

```diff
@@ -106,6 +106,7 @@ Vagrant.configure("2") do |config|
       # Only execute once the Ansible provisioner,
       # when all the machines are up and ready.
       if i == $num_instances
+        config.vm.provision :shell, inline: '[[ -x "$(command -v yum)" ]] && ifup eth1', privileged: true
```

Comment:
This appears to be an issue impacting [others](https://github.com/mitchellh/vagrant/issues/8166), not just myself.  Other people on my team are also suffering from this problem.

```
Vagrant 1.9.1
VirtualBox Version 5.0.32 r112930
```

```
[jodewey:~/git/kargo] master+ ± vagrant box list  | grep centos/7
centos/7            (virtualbox, 1611.01)

[jodewey:~/git/kargo] master+ ± grep centos vagrant/config.rb
$box = "centos/7"
```

```
[jodewey:~/git/kargo] master+ ± vagrant up --no-provision
Bringing machine 'k8s-01' up with 'virtualbox' provider...
==> k8s-01: Importing base box 'centos/7'...
...

[jodewey:~/git/kargo] master+ ± vagrant ssh -c 'ip addr s eth1'
3: eth1: <BROADCAST,MULTICAST> mtu 1500 qdisc pfifo_fast state DOWN qlen 1000
    link/ether 08:00:27:78:67:57 brd ff:ff:ff:ff:ff:ff
Connection to 127.0.0.1 closed.
```



This change seems fairly straight forward until vagrant gets it's act together.  Doesn't impact any other distro but the ones having this particular issue.  I'm happy to remove this patch, once Vagrant corrects the issue.